app/components/Inputs/CheckBox/index.jsx

```diff
@@ -0,0 +1,47 @@
+// flow
```

Comment:
For consistency with the rest of the application (I ignored this in your contribution with the Send work but would like to address it in the future) can we call this file CheckBox.jsx with another file called index.js that exports the component as well as everywhere else in this PR where this pattern exists

Reply:
No problem.